src/App.js

```diff
@@ -1,29 +1,12 @@
-import logo from './logo.svg';
+/* eslint-disable jsx-quotes */
```

Comment:
- [ ] Please, consider removing all linter disables - let the linter makes its direct job, aka help you find bugs in the early stages of development 👍